src/assets/styles/styles/custom/_button.scss

```diff
@@ -0,0 +1,33 @@
+.btn {
+  background: none;
+  border: none;
+  padding: 5px 20px 5px;
+  border-radius: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial,
+    sans-serif;
+  font-size: 15px;
```

Comment:
Kilka rzeczy nie jest zgodne z tym co mamy na Figmie:
![image](https://user-images.githubusercontent.com/23723122/100566489-7ef79400-32c6-11eb-98c9-047c0dfb7088.png)
Używamy czcionki Roboto, font ma być 16px, a jego waga to 400.

Reply:
Dodatkowo mam prośbę, żebyś część rzeczy wyniosła do pliku z globalnymi stałymi: [_variables.scss](https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/blob/develop/src/assets/styles/styles/custom/_variables.scss). Chodzi mi o:
1. btn-font-size-lg
2. btn-font-weight
3. btn-round-radius
4. font-family-base
Możliwe, że są jeszcze jakieś inne, których używasz, a je pominąłem. Jeśli jeszcze coś znajdziesz to oczywiście podmieniaj :D

Reply:
Ojej, przepraszam, nie wiedziałam: patrzyłam na Figmę bez zakładania konta i wtedy do dyspozycji jest tylko całościowy widok. Ale zaraz to naprawię.